engine/engine.go

```diff
@@ -230,5 +230,47 @@ func cmd() *cobra.Command {
 		},
 	})
 
+	cmd.AddCommand(&cobra.Command{
+		Use:   "sign-tls-cert [public-key-file] [OPTIONAL output-file]",
+		Short: "Creates and signs a certificate for usage as TLS client certificate",
+		Long: "Creates a CSR for the given public-key using a TLS client certificate template. " +
```

Comment:
formatting is off I think (IntelliJ normally lines out values in struct initilization)

Reply:
I used `gofmt -w .`